app/javascript/components/GuitarGallery.js

```diff
@@ -0,0 +1,57 @@
+import React, { useEffect, useState } from 'react';
+import axios from 'axios';
+import { Title, TitleSizes, Gallery } from '@patternfly/react-core';
+import GuitarInfo from './GuitarInfo';
+import guitarsUrl from './constants';
```

Comment:
It's a const, should be imported accordingly :)

Reply:
Fixed 👍🏼